manager/src/test/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPITest.java

```diff
@@ -248,6 +248,9 @@ public void testGetBridgesToDelete() {
 
         BridgeDTO bridge = bridgesToDeployOrDelete.get(0);
 
+        bridge.setStatus(ManagedResourceStatus.READY);
```

Comment:
The `Bridge` should be in `READY` state after [this](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/c6eff16620815d47f74ada6483ad18d14256ca20/manager/src/test/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPITest.java#L242) completes.

Reply:
Here we just create a bridge which set it to `APPROVED` state and then try to delete it. that why I set to `READY` manually so that we could test delete funcntionality.